django/contrib/admin/utils.py

```diff
@@ -222,6 +222,11 @@ def nested(self, format_callback=None):
         roots = []
         for root in self.edges.get(None, ()):
             roots.extend(self._nested(root, seen, format_callback))
+        # All values are not stored by key None in dict self.edges
+        for key in self.edges:
+            if str(key) != "None":
```

Comment:
Because items with "None" key are accessed above by self.edges.get(None,()). So now i only need to access those items whose key are not "None".